policyengine_us/parameters/gov/states/mi/tax/income/credits/home_heating_credit/alternate_credit/alternate_credit_rate.yaml

```diff
@@ -0,0 +1,12 @@
+##add reference for 2021
+description: Multiply line_44 by this rate for alternate credit amount.
+values:
+  2021-01-01: 0.7
+metadata:
+  period: year
+  unit: /1
+  label: total_household_resources_rate
+  reference:
+  - title: 2022 MICHIGAN Home Heating Credit Claim MI-1040CR-7
+    href: https://www.michigan.gov/taxes/-/media/Project/Websites/taxes/Forms/2022/2022-IIT-Forms/MI-1040CR7.pdf?rev=c36e54deae424566bb6abd1cbe8af7cb&hash=B39BA9887DD31395E754CC0641EFB512
+
```

Comment:
we need to add the legal reference as well as the 2021 tax form - a total of 3 references for each yaml file